tests/integration/test_yaml_configuration/configs/clickhouse_main_xml/config.d/more_clusters.yaml

```diff
@@ -0,0 +1,24 @@
+yandex:
+  remote_servers:
+    __cdata: "\r\nYou can run additional servers simply as\r\n./clickhouse-server -- --path=9001 --tcp_port=9001\r\n"
```

Comment:
Is `__cdata` it really necessarily?
Also it looks like a comment, should it be translated into comment?

Reply:
Hmm, I copied config files directly from programs/server, so it seems that some configs are way too detailed for a test. I'll try fix it in the next commit.

Reply:
Checked the original file, the cdata string is a comment. I'll fix it in the next commit.